<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReconnectInfoDialog</class>
 <widget class="QDialog" name="ReconnectInfoDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reconnecting...</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Please wait while the connection is being restored...</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1">
        <item row="0" column="2">
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>Server name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QnProgressWidget" name="progressWidget" native="true"/>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="iconLabel">
          <property name="text">
           <string notr="true">&lt;Icon&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QnProgressWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/common/progress_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
